ledfx/effects/plasma2d.py

```diff
@@ -107,10 +107,8 @@ def draw(self):
         if self.test:
             self.draw_test(self.m_draw)
 
-        current_time = timeit.default_timer() - self.start_time
-
         plasma_array = self.generate_plasma(
-            self.r_width, self.r_height, current_time, self.bar
+            self.r_width, self.r_height, self.current_time, self.bar
```

Comment:
I will need to look if this has the same implication, if its just a time continuum which I think it is, this is fine. My original code is a little kludgy from initial implementation.

Reply:
I think this is fine and gets rid of a use of self.start_time